docs/wiki/99_Developers/Developers_Guide.md

```diff
@@ -0,0 +1,241 @@
+
+**Labels:** On the right side of the browser, you will see the option to add labels. Please add any that are related to the PR. 
+**If** you are working on release testing, make sure to include the label (ex. 23.0.0-testing) to the PR.
+
+**Reviewers/Assignees:** Reviewers are people that you would like to ping specifically, but are not the same as assignees.
+Be sure to consult the Contributing and Code Review guidelines. For example, 2 reviewers must approve the PR and any code change must be pulled and tested by a developer before a PR is marked as `Passed Manual Tests`. 
+
+**Travis:** The Travis build status and results are linked near the bottom of your PR in GitHub. 
+You can restart Travis if it gives error messages that seem inappropriate, or if the build stalls (for longer than 1h).
+if you are pushing a last typo fix on a PR, it is okay to add `[skip ci]` in the commit message. It is not okay to do this if it's the first commit or if you are fixing more than documentation typos.
+
+**Documentation PRs:** If working specifically on markdown files or other documentation, be sure to check the actual look of the document once you have pushed your PR and fix any formatting errors. You can do this by going to “Files Changed” and viewing how your markdown file will actually appear in GitHub. Don’t forget to click all links. 
+
+## 4. Code Review
+
+### Testing or Contributing to someone's PR
+First, make a remote of their fork:
+
+    git remote add "their_fork" <their_fork_url>
+          
+Then, fetch their remote fork and checkout the branch related to their PR:
+
+    git fetch their_fork
+    git checkout their_fork/branch_name
+
+Alternatively, to avoid being under the detached HEAD state, checkout and create a local branch with the command
+
+    git checkout -b branch their_fork/branch_name
+
+Follow the provided testing instructions to test their code on your VM. 
+
+If you made changes (with their permission), commit and push them to that branch:
+
+    git commit -m "Changes made"
+    git push their_fork branch_name
+
+Don’t forget to consult the Links (see above) for coding review standards and guidelines and the code review checklist.
+
+### Labels
+The `Passed Manual Tests` label can be added to a PR if a reviewer has tested the code on their own VM and determined that it performs as intended.
+
+The `Needs Rebase` label can be added if a PR has merge conflicts with the branch it is based off of.
+
+The `Needs Work` label can be added if a PR is not yet ready for review and requires more work before it can be approved.
+
+Other Labels can also be added by reviewers at their discretion. 
+
+### Code Review Etiquette
+The most important thing to remember during code review is to be respectful and positive in all correspondence with other developers. It is also good practice
+to be on top of all comments or requests and to respond to every comment.
+
+#### When to ask for or remind someone to do a review
```

Comment:
significant tone update needed if this is going to be published.  please move to internal doc.